js/packages/screenshoter/test/e2e/web/hover-element.spec.js

```diff
@@ -0,0 +1,27 @@
+const {makeDriver, test, logger} = require('../e2e')
+
+describe('screenshoter web', () => {
+  let driver, destroyDriver
+
+  before(async () => {
+    ;[driver, destroyDriver] = await makeDriver({type: 'chrome', logger})
+    await driver.visit('https://applitools.github.io/demo/TestPages/StickyHeaderWithRegions')
+  })
+
+  after(async () => {
+    await destroyDriver()
+  })
+
+  it('take element in frame screenshot with hover element', async () => {
+    const element = await driver.element('#input')
+    await driver.target.$(element.target).moveTo()
+
+    await test({
+      type: 'web',
+      tag: 'hover-element',
```

Comment:
The convention here is 
```
`${what's on the screenshot (page | frame | element)}-${fully or not}-${what's special about it}`
```

```suggestion
      tag: 'element-hovered',
```

Reply:
The same for file name